### ISSUE TITLE
Device do not show up as unknown devices once Deleted

### ISSUE THREAD

**Issue by Tamburasca** [gotthardp/lorawan-server#637]
We see the strange behavior that devices do not show up in the dashboard anymore once they were deleted from the activated nodes. Is this desired?

Cheerio, tamburasca

**Comment by gotthardp** [comment]
Where would you expect the node to show after being deleted?

**Comment by Tamburasca** [comment]
![image](https://user-images.githubusercontent.com/33293848/57915875-ff47d700-7891-11e9-8c10-5458d5d67d0b.png)

here as red bar - unknown device

**Comment by gotthardp** [comment]
Yeah, that would be logical, but it doesn't work that way. The frames are deleted and don't show as "unknown".

**Comment by Tamburasca** [comment]
And this applies also in case the gateway still sees the devices

**Comment by gotthardp** [comment]
When a new frame from the deleted node arrives, it should be displayed as "unknown" (if logging of unknown frames is enabled)

**Comment by Tamburasca** [comment]
good to know, I haven't gotten that far yet! Thanks.

**Comment by Tamburasca** [comment]
I trawled through your documentation, however, did not find how to enable unknown frames. Sorry for being persistent.

**Comment by gotthardp** [comment]
Under Area, there is "Log Ignored"
